main.js

```diff
@@ -0,0 +1,44 @@
+(function () {
+    'use strict';
+
+    function get(id) {
```

Comment:
let's use ES6 instead of ES5 (you can read about it a bit here: https://www.geeksforgeeks.org/difference-between-es5-and-es6/)

The syntax would be:

```
const get = (id) => {
    return document.getElementById(id);
}
```

Same comment would apply to the countdown function

Reply:
I'm on the fence about the name of this function. `get` is very arbitrary and doesn't explain what the function does. At the same time, the code is very clean later on (e.g. `get("minutes")`)

I'll leave it up to you if you want to rename this function to something more explicit/specific, just sharing my thoughts as this is the type of feedback you might encounter as you go.